tests/rustdoc-ui/z-help.stdout

```diff
@@ -98,6 +98,7 @@
     -Z      mir-pretty-relative-line-numbers=val -- use line numbers relative to the function in mir pretty printing
     -Z                       move-size-limit=val -- the size at which the `large_assignments` lint starts to be emitted
     -Z                       mutable-noalias=val -- emit noalias metadata for mutable references (default: yes)
+    -Z                            new-rpitit=val -- specify to use the new rpitit system that lowers return position impl trait in trait as associated types both in the trait and in the corresponding impl
```

Comment:
I'm not a fan of this `-Z` flag name. Maybe something like `-Z lower-impl-trait-in-trait-to-assoc-ty`? It's verbose but clearer.

And the description could be:

"modify the lowering strategy for `impl Trait` in traits so that they are lowered to generic associated types"

Reply:
Done